### ISSUE TITLE
Better name / alias for `fct_reorder2()`

### ISSUE THREAD

**Issue by jennybc** [tidyverse/forcats#264]
I'm a huge fan of `fct_reorder2()`, for the reasons highlighted here:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">If you make a line plot, make the legend order match some notion of the data order (or use direct labelling). Linked gist shows how to do this in <a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a> with forcats::fct_reorder2().<a href="https://t.co/1IHbG7wIG8">https://t.co/1IHbG7wIG8</a> <a href="https://t.co/hadqsmfLO6">pic.twitter.com/hadqsmfLO6</a></p>&mdash; Jenny Bryan (@JennyBryan) <a href="https://twitter.com/JennyBryan/status/1244499728831672320?ref_src=twsrc%5Etfw">March 30, 2020</a></blockquote> 

Gist: <https://gist.github.com/jennybc/847c6b43c4e35cec2e5bb30a3f38af73>

![image](https://user-images.githubusercontent.com/599454/85338141-979d8b80-b496-11ea-971f-9663d83d4407.png)

Maybe the under-appreciation for `fct_reorder2()` is partially due to its name? It doesn't really evoke what the function does.

I don't have a great alternative but something along the lines of `fct_reorder_for_legend()`?

**Comment by ahcyip** [comment]
Agreed, and I believe the names of the functions fct_relevel, fct_recode, fct_reorder all need some re-consideration.
https://github.com/tidyverse/forcats/issues/269

**Comment by hadley** [comment]
If it helps, you can think of the 2d as "2d"
